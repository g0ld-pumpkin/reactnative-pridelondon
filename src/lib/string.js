// @flow
/* eslint-disable import/prefer-default-export */

export const hyphenate = (text: string): string =>
  text
    .toLowerCase()
    .split(" ")
    .join("-");
