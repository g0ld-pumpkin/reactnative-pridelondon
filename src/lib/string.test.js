import { hyphenate } from "./string";

describe("hyphenate", () => {
  it("formats a string into lowercase with hyphens in between words", () => {
    expect(hyphenate("Hello Jolly World")).toMatchSnapshot();
  });
});
