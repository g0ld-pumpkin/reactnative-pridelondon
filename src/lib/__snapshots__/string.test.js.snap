// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hyphenate formats a string into lowercase with hyphens in between words 1`] = `"hello-jolly-world"`;
